src/amo/components/HeadMetaTags/index.js

```diff
@@ -0,0 +1,88 @@
+/* @flow */
+import config from 'config';
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import { getCanonicalURL } from 'amo/utils';
+import type { AppState } from 'amo/store';
+
+type Props = {|
+  date?: Date | null,
+  description: string,
+  image?: string | null,
+  lastModified?: Date | null,
+  title: string,
+|};
+
+type InternalProps = {|
+  ...Props,
+  _config: typeof config,
+  lang: string,
+  locationPathname: string,
+|};
+
+export class HeadMetaTagsBase extends React.PureComponent<InternalProps> {
+  renderOpenGraph() {
+    const {
+      _config,
+      description,
+      image,
+      lang,
+      locationPathname,
+      title,
+    } = this.props;
+
+    const tags = [
+      <meta key="og:type" property="og:type" content="website" />,
+      <meta
+        key="og:url"
+        property="og:url"
+        content={getCanonicalURL({ _config, locationPathname })}
```

Comment:
This used to be simply `addon.url`. Just want to confirm that this change is intended.